### DISCUSSION TITLE
Disconnected from Manager error

### DISCUSSION THREAD

**Discussion by bisket82** [Security-Onion-Solutions/securityonion#11526] [2.4]
### Version

2.4.0

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

16

### RAM

32

### Storage for /

75GB

### Storage for /nsm

235GB

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

I'm getting a "Disconnected from Manager" error. All services are running and I have restarted the system three time with no luck.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment]
Is there a firewall, proxy, or other network device between your browser and your Security Onion machine that may be interfering with the traffic?

What browser are you using?

Have you tried using incognito mode in your browser?

Have you tried a different browser?

Have you tried a different machine?

**Comment by StefanSa** [comment]
I also had this error after installing the last hotfix. Clearing the browser cache helped me.
